README.md

```diff
@@ -1 +1,63 @@
-# Software Product Health Assistant
\ No newline at end of file
+![SPHA Logo](docs/img/Software_Project_Health_Assistant_Secondary-Logo.png)
+
+## About
+
+SPHA is a fully automated tool suite that assesses and communicates all aspects
+of software product quality. It does so by combining data about your projects
+from sources like ticketing systems, and static analysis tools. For more details
+see [software-product.health](https://www.software-product.health).
+
+## SPHA-CLI Tool
+
+This project contains SPHA's command line tool, a wrapper around our
+[core library](https://www.github.com/fraunhofer-iem/spha). The CLI-tool can `transform` tool results
+into `RawValueKpis`, `calculate` a given `KpiHierarchy` based on `RawValueKpis`, and
+generate a human-readable report.  
+A tool demo using our [GitHub Action](https://www.github.com/fraunhofer-iem/spha-action) can be
+found [here](https://www.github.com/fraunhofer-iem/spha-demo).
+
+## Installation
+
+### Using Docker
+
+The easiest way to get started with SPHA is by using the included docker image. To
+build it locally run `docker build -t TAG_NAME .`. Afterward, you can run the container
```

Comment:
- is it really  `docker build -t TAG_NAME .`? (with the period)
- Afterwards instead of aftaward?

Reply:
Yep. It builds the file in the Current folder, indicated by the `.`